os/booting-with-libvirt.md

```diff
@@ -71,9 +71,7 @@ mkdir -p /var/lib/libvirt/images/coreos/coreos1/openstack/latest
 touch /var/lib/libvirt/images/coreos/coreos1/openstack/latest/user_data
 ```
 
-The `user_data` file may contain a script for a [cloud config][cloud-config]
-file. We recommend using ssh keys to log into the VM so at a minimum the
-contents of `user_data` should look something like this:
+The `user_data` file declares machine configuration in the [cloud config]({{site.baseurl}}/docs/cluster-management/setup/cloudinit-cloud-config) format. We recommend using ssh keys to log into the VM, and since those keys are stored in `user_data,` at minimum that file should contain something like this:
```

Comment:
I think we should probably go with a direct link to `https://coreos.com/os/docs/latest/cloud-config.html` since our rewriter isn't smart enough to know that these two repos get merged together.

If we just did `cloud-config.html` as a relative link, it wouldn't work on Github...